allure/utils.py

```diff
@@ -131,8 +131,11 @@ def get_exception_message(report):
     """
     get exception message from pytest's internal ``report`` object
     """
-    return (getattr(report, 'exception', None) and present_exception(report.exception.value)) or (hasattr(report, 'result') and report.result) or report.outcome
-
+    return (getattr(report, 'exception', None) and present_exception(report.exception.value)) or \
```

Comment:
according to https://bitbucket.org/pytest-dev/pytest/src/7ed701fa2fb554bfc0618d447dfec700cc697407/_pytest/skipping.py?at=2.7.0#cl-182